analysis_options.yaml

```diff
@@ -45,4 +45,14 @@ linter:
 
     join_return_with_assignment: false
 
-    prefer_const_literals_to_create_immutables: false
\ No newline at end of file
+    prefer_const_literals_to_create_immutables: false
+
+
+    ####################################################
+    #### Added to make things work for now. Must be ####
+    ############# removed in future PRs ################
+    ####################################################
+
+    depend_on_referenced_packages: false
+    avoid_dynamic_calls: false
```

Comment:
Does this need to be reverted to `true` after the merge? If so, add a comment stating this.

Reply:
@palisadoes I have already added suitable comments in the lint suppressions on how to turn them on and resolve the subsequent issues as in the below image

![image](https://user-images.githubusercontent.com/62198564/219885662-ab8dad3c-9baa-4e11-a002-64ba6615e12c.png)